keyboards/handwired/sticc14/readme.md

```diff
@@ -0,0 +1,14 @@
+# Sticc14
+
+![Sticc14](https://i.imgur.com/blzFw5R.jpg)
+
+Sticc14 is a simple macropad, designed to be easy to build and affordable.
+
+Keyboard Maintainer: [ErkHal](https://github.com/erkhal)  
+Hardware Supported: Sticc14  
+Hardware Availability: [Ergorius](https://reddit.com/u/ergorius)  
+Make example for this keyboard (after setting up your build environment):
+
+    make Sticc14:default
```

Comment:
This needs to be a case-sensitive match to the path relative to `qmk_firmware/keyboards/`, otherwise `make` won't work.

```suggestion
    make handwired/sticc14:default
```